app/routes/poll-network.tsx

```diff
@@ -0,0 +1,50 @@
+import { useFetcher, useLoaderData } from "@remix-run/react"
+import { useEffect, useState } from "react"
+import { fetchNetworkStatus } from "~/cms/utils/fetch-network-status"
+import { StatuspageApiResponse } from "~/ui/design-system/src/lib/interfaces"
+
+export async function loader() {
+  return await fetchNetworkStatus()
+}
```

Comment:
i think it would probably be better to return an object here, e.g.

```
const networkStatus = await fetchNetworkStatus()
return {
  networkStatus
}
```